README.md

```diff
@@ -1,24 +1,73 @@
-# README
+# Blog
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Blog app is an example of a blog website. It's a functional website that shows the list of posts and enable users to interact with it by adding comments and liking posts.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get a local copy up and running follow these simple example steps.
 
-* Database creation
+git clone https://github.com/ishimwezachee/Blog.git
 
-* Database initialization
+cd Blog
 
-* How to run the test suite
+### Prerequisites
 
-* Services (job queues, cache servers, search engines, etc.)
+Ruby
+Rails
+PostgresSQL
 
-* Deployment instructions
+### Setup
 
-* ...
+Install gems with:
+
+```
+bundle install
+```
+
+Setup database with:
+
+```
+rails db:create
+rails db:migrate
+```
+
+### Usage
+
+Start server with:
+
+```
+rails s
+```
+
+Open http://localhost:3000/ in your browser.
+
+## Author
+
+👤 **[ishimwezachee](https://github.com/ishimwezachee)**
+
+- GitHub: [@zachee-ishimwe](https://github.com/ishimwezachee)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zachee-ishimwe/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ishimwezachee/Blog/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- A special thanks to Microverse for inspiring this project. 
+
+## 📝 License
+
+This project is [MIT](./LICENCE) licensed.
```

Comment:
- Please fix the MIT link for a completely professional repo

Reply:
Got it, thank you for the reviewer